awx/ui_next/src/screens/Template/JobTemplateAdd/JobTemplateAdd.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { mountWithContexts } from '@testUtils/enzymeHelpers';
+import JobTemplateAdd from './JobTemplateAdd';
+
+jest.mock('@api');
+
+describe('<JobTemplateAdd />', () => {
+  const defaultProps = {
+    description: '',
+    inventory: 0,
+    job_type: 'run',
+    name: '',
+    playbook: '',
+    project: 0,
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('should render Job Template Form', () => {
+    const wrapper = mountWithContexts(<JobTemplateAdd />);
+    expect(wrapper.find('JobTemplateForm').length).toBe(1);
+  });
+
+  test('should render Job Template Form with default values', () => {
```

Comment:
i'm not sure this test is really needed...we're sort of testing default props react code?

Reply:
I see where you're coming from. That said, I do see _some_ value in adding an additional layer of redundancy by checking for specific default values - default values of form inputs seem like the kind of thing where unexpected, subtle changes can happen unintentionally. By memorializing those default values here in this test, we can be absolutely certain that changes to those defaults are always intentional.

Reply:
update: talked about this one offline with @jlmitch5 

To clarify, I'm endorsing the _goal_ of this test, which is to check that the initialized default values for the field are what we expect. That said, I'd prefer if we avoided making assertions on props / states when we can, as that can make tests more brittle. 

Instead of making assertions on prop / state values, we could probably check the actual input values on the elements themselves.